### PR TITLE
Product of count of bits of each element in array is added

### PR DESCRIPTION
**Pull request by shivammahajancse**
### Have you read the [Contributing Guidelines on Pull Requests](https://github.com/TesseractCoding/NeoAlgo/blob/master/CONTRIBUTING.md#pull-requests)?

(Yes)

### Checklist

- [x] I've read the contribution guidelines.
- [x] I've checked the issue list before deciding what to submit.
- [x] I've edited the `README.md` and link to my code.

## Related Issues or Pull Requests

(#256 )

### CODE REVIEW

**Comment by shivammahajancse** [comment]
Please review sir @HarshCasper @iamrajiv @ankitaggarwal23 @atarax665 @gat786

File: docs/mathematics/product-of-count-of-bits-of-each-elements.mdx [hunk 1]
```diff
@@ -0,0 +1,174 @@
+---
+title: Product of count of bits of each element in array
+slug: /mathematics/product-of-count-of-bits-of-each-elements
+---
+
+## Introduction
+
+In this article, we will see how to calculate product of count of bits of each element present in array.
```
Comment (iamrajiv) [inline docs/mathematics/product-of-count-of-bits-of-each-elements.mdx:8]:
```suggestion
In this article, we will see how to calculate the product of the count of bits of each element present in the array.  
```

File: docs/mathematics/product-of-count-of-bits-of-each-elements.mdx [hunk 2]
```diff
@@ -0,0 +1,174 @@
...
+title: Product of count of bits of each element in array
+slug: /mathematics/product-of-count-of-bits-of-each-elements
+---
+
+## Introduction
+
+In this article, we will see how to calculate product of count of bits of each element present in array. 
+
+## Explanation 
+In this we are given an array. We have to calculate the product of bits of each element present in array for that first we have to calculate the number of bits in each element then simply multiply them.
```
Comment (iamrajiv) [inline docs/mathematics/product-of-count-of-bits-of-each-elements.mdx:11]:
```suggestion
In this, we are given an array. We have to calculate the product of bits of each element present in the array so first, we have to calculate the number of bits in each element then simply multiply them.
```

**Review by iamrajiv — CHANGES_REQUESTED** [review]
@shivammahajancse Implement the suggested changes.

**Comment by shivammahajancse** [comment]
All changes done sir @iamrajiv

**Review by iamrajiv — APPROVED** [review]

File: docs/mathematics/product-of-count-of-bits-of-each-elements.mdx [hunk 3]
```diff
@@ -0,0 +1,174 @@
+---
+title: Product of count of bits of each element in array
+slug: /mathematics/product-of-count-of-bits-of-each-elements
+---
+
+## Introduction
+
+In this article, we will see how to calculate the product of the count of bits of each element present in the array.  
+
+## Explanation
```
Comment (atarax665) [inline docs/mathematics/product-of-count-of-bits-of-each-elements.mdx:10]: Add a line break after this.

File: docs/mathematics/product-of-count-of-bits-of-each-elements.mdx [hunk 4]
```diff
@@ -0,0 +1,174 @@
...
+slug: /mathematics/product-of-count-of-bits-of-each-elements
+---
+
+## Introduction
+
+In this article, we will see how to calculate the product of the count of bits of each element present in the array.  
+
+## Explanation 
+In this, we are given an array. We have to calculate the product of bits of each element present in the array so first, we have to calculate the number of bits in each element then simply multiply them.
+#### Example 1
```
Comment (atarax665) [inline docs/mathematics/product-of-count-of-bits-of-each-elements.mdx:12]:
```suggestion
#### Example :
```

**Review by atarax665 — CHANGES_REQUESTED** [review]

**Comment by shivammahajancse** [comment]
Changes done sir @atarax665

**Review by atarax665 — APPROVED** [review]

**Pull request merged**